lib/manager/npm/post-update/npm.ts

```diff
@@ -113,7 +118,9 @@ export async function generateLockFile(
     // massage to shrinkwrap if necessary
     if (
       filename === 'npm-shrinkwrap.json' &&
-      (await pathExists(upath.join(cwd, 'package-lock.json')))
+      (await localPathExists(
+        lockFileName.replace('npm-shrinkwrap.json', 'package-lock.json')
```

Comment:
```suggestion
        lockFileName.replace(regex('(?:^|\/)npm-shrinkwrap\.json$'), 'package-lock.json')
```
We should use regex to be sure we don't wrongly replace other parts of path. We never know what people do. 😉